GitFyle.Core.Api/Brokers/Storages/StorageBroker.Contributions.cs

```diff
@@ -13,6 +13,9 @@ internal partial class StorageBroker
     {
         public DbSet<Contribution> Contributions { get; set; }
 
+        public async ValueTask<Contribution> InsertContributionAsync(Contribution contribution) =>
+            await InsertAsync(contribution);
```

Comment:
if we are making it generic then on line 17, why not InsertAsync<Contribution> ?

Reply:
Isn't it InsertAsync?